Makefile

```diff
@@ -173,7 +176,7 @@ systemtray: $(ADDON_ASSET_FILE)
 	go install -tags "$(BUILD_TAGS)" -pkgdir=$(ADDON_BINDATA_DIR) -ldflags="$(VERSION_VARIABLES)" ./cmd/minishift
 
 .PHONY: cross_systemtray
-cross_systemtray: clean $(BUILD_DIR)/darwin-amd64/systemtray/minishift $(BUILD_DIR)/windows-amd64/systemtray/minishift.exe
+cross_systemtray: clean $(BUILD_DIR)/windows-amd64/systemtray/minishift.exe $(BUILD_DIR)/darwin-amd64/systemtray/minishift 
```

Comment:
@anjannath does changing the sequence needed here?

Reply:
Nope.. it is not needed, this was a result of my testing, i wanted to comment out the darwin part, so moved it towards the end. The changes got left